covid_assembly_artic.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+ 
+#SBATCH --nodes=1 
+#SBATCH --ntasks=1 
+#SBATCH --cpus-per-task=10 
+#SBATCH --mem=50000 # using 50Gb of memory
+#SBATCH --error covid_assembly-%A-%a.err   #sam_to_fastq-%A-%a.err 
+#SBATCH --output covid_assembly-%A-%a.out  #sam_to_fastq-%A-%a.out
+#SBATCH --mail-type=END,FAIL
+#SBATCH --mail-user=kmane@mrc.gm
+###SBATCH --time=5-00:00:00
+
+typeset -F SECONDS
+module load artic
+threads=10
```

Comment:
I'm not sure this script needs to be included in the package - if you want to provide an example of an HPC cluster job script, it might be better to put it under `inst/`?